core/src/main/kotlin/me/lazmaid/kraph/lang/Variables.kt

```diff
@@ -1,10 +1,22 @@
 package me.lazmaid.kraph.lang
 
+import me.lazmaid.kraph.KraphVariable
+
 /**
  * Created by OinkIguana on 10/25/2017 AD.
  */
 
-// TODO: implement variables
-internal class Variables() {
-    fun print(): String? = null
+internal class Variables {
```

Comment:
All the node should implement `GraphQLNode`. So, it would be easier to to handle formatting logic

Reply:
I think it would be confusing if Variables implements GraphQLNode, since the parameters of print (format: PrintFormat, previousLevel: Int) don't make any sense.

Reply:
At this moment, It may not make sense much to make it implement `GraphQLNode`, but it will make it more easier if we need to implement some features that apply to all the node in the Kraph